src/Lykk.Service.KeyGenerator/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Lykk.Service.Stellar.KeyGenerator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //var stellarPrivateKeyPair = StellarBase.KeyPair.Random();
```

Comment:
What does this program do?

Reply:
This program just generates private key/ public address.
Applied fixes